iOS Example Frame/iOS Example Frame/AppDelegate.swift

```diff
@@ -14,6 +14,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        UIFont.loadAllCheckoutFonts
```

Comment:
will the merchants have to call this from their app?

Reply:
No, just for us internally.